_posts/2023-04-29-release.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Release 2023.17
+author: Priya
+date: 2023-04-29
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Email Notification for Pipeline Errors.(an alpha feature on demand)
+* Mammoth have added an email notification feature to alert users when a task or action fails during pipeline execution.
```

Comment:
Mammoth now provides email alerts to users when a task fails during pipeline execution